3_parameter_efficient_finetuning/prompt_tuning.md

```diff
@@ -51,15 +53,22 @@ When implementing prompt tuning, start with a small number of virtual tokens (8-
 
 Training requires slightly different considerations than full fine-tuning. Higher learning rates often work well, but careful monitoring of prompt token gradients is essential. Regular validation on diverse examples helps ensure robust performance across different scenarios.
 
-## Applications
+## Application
+
+Prompt tuning excels in several scenarios:
+
+1. Multi-task deployment
+2. Resource-constrained environments
+3. Rapid task adaptation
+4. Privacy-sensitive applications
 
-Prompt tuning excels in several scenarios, particularly for classification tasks and simple generation tasks. Its minimal resource requirements make it ideal for environments with limited computational resources. The ability to quickly switch between tasks also makes it valuable for multi-task applications where different behaviors are needed at different times.
+As models get smaller, prompt tuning becomes less competitive compared to full fine-tuning. For example, on models like SmolLM2 scales prompt tuning is less relevant than full fine-tuning. 
 
 ## Next Steps
 
-To get hands-on experience with prompt tuning, try the [Prompt Tuning Tutorial](./notebooks/prompt_tuning_example.ipynb). This practical guide will walk you through implementing the technique with your own model and data.
```

Comment:
is there not going to be a promt_tuning_example anymore?

Reply:
Nope 😢